formats/email_alert_signup/publisher/details.json

```diff
@@ -13,17 +13,7 @@
       "minProperties": 1,
       "properties": {
         "tags": {
-          "type": "object",
-          "description": "DEPRECATED: The tags used to match subscribers lists",
```

Comment:
Oh.  Seems that "tags" were removed from the hmrc-manuals-api that translates from the hmrc publisher and publishing-api so maybe we can drop tags from that schema entirely.  I think the "DEPRECATED" tag might still be useful, although it's more about the `details/tags` key than the definition per se.